### DISCUSSION TITLE
Getting weird duration on the Alerts

### DISCUSSION THREAD

**Discussion by eric-moreira** [netdata/netdata#17130] [Q&A]
Hello I'm using netdata on some servers on my company.
Recently we started receiving Slack alerts with weird durations:
254 years, -70 years, etc.

It happens on different servers, and they are all connected to the cloud. It seems that it started recently.

I would like some insights about where I should look for answers, or how can I reset the alerts duration.

**Comment by stelfrag** [comment]
Hi @eric-moreira , what version is your netdata agents?

**Reply by eric-moreira** [reply]
v1.44.0-477-g8d18e1597
v1.44.3

Some of them are different,

**Reply by stelfrag** [reply]
Could you share the space id that has those problematic alerts? You can find the space id in space settings and info section.

You can email me the space id at stelios@netdata.cloud

**Reply by eric-moreira** [reply]
Sent you! Thanks for the headsup.

**Reply by eric-moreira** [reply]
Do you have any insights on this?

**Reply by stelfrag** [reply]
Have you received notifications with these strange duration after Jan 13th? We did notice the last notification with "weird" duration was on Jan 13th. There has been a fix already applied on Feb 12th, 2024

**Reply by eric-moreira** [reply]
We are still receiving weird durations.
from today: 
```
Alert:
used_swap
Chart:
mem.swap
Context:
mem.swap
Class:
Utilization
Duration:
-271 years
```

**Reply by stelfrag** [reply]
Is this on the same space id you shared?

**Reply by eric-moreira** [reply]
Yes it is.

**Reply by stelfrag** [reply]
Thank you -- We are investigating...

**Reply by stelfrag** [reply]
> Thank you -- We are investigating...

A new fix has been applied. If you can confirm that it works for you as expected, we can consider the issue resolved. Thank you for your patience while we tracked this down
